bin/install_openstackbmc.sh

```diff
@@ -45,12 +45,25 @@ export OS_TENANT_NAME=$os_tenant
 export OS_PASSWORD=$os_password
 export OS_AUTH_URL=$os_auth_url
 export OS_PROJECT_NAME=$os_project
-export OS_USER_DOMAIN_ID=$os__user_domain
-export OS_PROJECT_DOMAIN_ID=$os__project_domain
+export OS_USER_DOMAIN_ID=$os_user_domain
```

Comment:
Ok i got it why we cannot use ${variable} because these are not just SHELL variables, instead they are substituted using str_replace in heat templates.